spec/uploader/callback_spec.rb

```diff
@@ -3,19 +3,27 @@
 describe CarrierWave::Uploader do
 
   it "should keep callbacks on different classes isolated" do
+    default_before_callbacks = [
+      :check_whitelist!,
```

Comment:
I actually did it already :bowtie:
https://github.com/mehlah/carrierwave/compare/remove-magicmime...mehlah:refactor-extensions-white/blacklist#diff-2bca1d8b1d06f3d29a6553468c7b940fR7
I'll PR the refactor for better consistency as soon as this is accepted.